### PR TITLE
Proposal/scalping overhaul

### PR DESCRIPTION
**Pull request by ACannamela**
https://www.realmsnet.net/proposals/94

### CODE REVIEW

File: Omnibus.md [hunk 1]
```diff
@@ -591,84 +592,97 @@ attack the unsuspecting looters. If you are unsure as to whether someone's
...
-since the scalped PC was scalped in order to raise them. A different spellcaster
-must provide each [Raise Dead](#raise-dead) spell used for this purpose.
+## 3.5: [Soul Loss]{.new} ## {#soul-loss}
+
+[Soul loss]{.new} is the removal of a character's life essence. It is a
+more serious form of death that requires more than a simple spell to repair.
+[A character who has lost their soul]{.new} cannot be raised or animated
+until their soul has been restored.
+
+### [Soul Token]{.new} ### {#soul-tokens}
```
Comment (pbobell) [inline Omnibus.md]: `### [Soul Tokens]{.new} ### {#soul-tokens}`

Comment (pbobell) [reply Omnibus.md]: Typo

File: Omnibus.md [hunk 2]
```diff
@@ -591,84 +592,97 @@ attack the unsuspecting looters. If you are unsure as to whether someone's
...
+
+A [soul token]{.new} is a token carried by each player, which represents
```
Comment (pbobell) [inline Omnibus.md]: `A [soul token]{.new} is a token carried by each player which represents`

File: Omnibus.md (hunk 1 above)
Comment (ACannamela) [reply Omnibus.md]: Change incorporated

File: Omnibus.md (hunk 2 above)
Comment (ACannamela) [reply Omnibus.md]: Change incorporated

File: Omnibus.md [hunk 3]
```diff
@@ -591,84 +592,97 @@ attack the unsuspecting looters. If you are unsure as to whether someone's
...
+the life essence of that player's character. Players must be carrying their PC's
+[soul token]{.new} at all times, unless the PC is dead and
+[soulless]{.new} (see below), or unless they are under the influence of a
```
Comment (pbobell) [inline Omnibus.md]: Other (see below) instances have been linked. Is that unnecessary since the section is immediately below? @3geek14

Comment (pbobell) [reply Omnibus.md]: Additionally:
`[soulless]{.new} (see below) or unless they are under the influence of a`

File: Omnibus.md [hunk 4]
```diff
@@ -591,84 +592,97 @@ attack the unsuspecting looters. If you are unsure as to whether someone's
...
+[Soul loss]{.new} is the removal of a character's life essence. It is a
+more serious form of death that requires more than a simple spell to repair.
+[A character who has lost their soul]{.new} cannot be raised or animated
+until their soul has been restored.
+
+### [Soul Token]{.new} ### {#soul-tokens}
+
+A [soul token]{.new} is a token carried by each player, which represents
+the life essence of that player's character. Players must be carrying their PC's
+[soul token]{.new} at all times, unless the PC is dead and
```
Comment (pbobell) [inline Omnibus.md]: `[soul token]{.new} at all times unless the PC is dead and`

File: Omnibus.md [hunk 5]
```diff
@@ -591,84 +592,97 @@ attack the unsuspecting looters. If you are unsure as to whether someone's
...
+[soulless]{.new} (see below), or unless they are under the influence of a
+spell which removes the PC's [soul. Soul tokens]{.new} must have their
+player's name and the PC's name written legibly on it. A [soul token]{.new}
```
Comment (pbobell) [inline Omnibus.md]: `player's name and the PC's name written legibly on them. A [soul token]{.new}`

File: Omnibus.md [hunk 6]
```diff
@@ -591,84 +592,97 @@ attack the unsuspecting looters. If you are unsure as to whether someone's
...
+does not exist physically in the IC world therefore it cannot be searched. A
+[soul token]{.new} is a non-magical and non-stealable marshaling tool.
+
+### [Destroying a Body]{.new} ### {#destroying-a-body}
+
+In order [for a player to separate a soul from a body, they must destroy
+it by]{.new} striking 200 blows with a weapon beside the body of the character
```
Comment (pbobell) [inline Omnibus.md]: `it by striking]{.new} 200 blows with a weapon beside the body of the character `

File: Omnibus.md [hunk 7]
```diff
@@ -591,84 +592,97 @@ attack the unsuspecting looters. If you are unsure as to whether someone's
...
+effectively divides the number of blows to be struck between the number of
+participants and number of weapons used. Some monsters and characters under
+certain spells may require more blows to completely [destroy]{.new}. If you
+strike 200 blows and the victim says "The job is not yet done," then the [body
+is not yet destroyed]{.new}. Some spells, such as
+[Strange Brew: Potion of Acid](#potion-of-acid) and
+[Assassin's Blade](#assassins-blade), may accelerate the act of [destroying a
+body]{.new}.
+
+After the [body is destroyed]{.new}, the character whose body has been
```
Comment (pbobell) [inline Omnibus.md]: `After the [body is destroyed]{.new}, [the character whose body has been`

File: Omnibus.md [hunk 8]
```diff
@@ -591,84 +592,97 @@ attack the unsuspecting looters. If you are unsure as to whether someone's
...
+destroyed is [rendered soulless]{.new}.  The player of the
```
Comment (pbobell) [inline Omnibus.md]: `destroyed is rendered soulless. T]{.new}he player of the`

File: Omnibus.md [hunk 9]
```diff
@@ -591,84 +592,97 @@ attack the unsuspecting looters. If you are unsure as to whether someone's
...
+[soulless]{.new} character must present their character's [soul]{.new}
```
Comment (pbobell) [inline Omnibus.md]: `[soulless]{.new} character must present their character's [soul`

File: Omnibus.md [hunk 10]
```diff
@@ -591,84 +592,97 @@ attack the unsuspecting looters. If you are unsure as to whether someone's
...
+token to the character that just [destroyed their body]{.new}. It is
```
Comment (pbobell) [inline Omnibus.md]: `token]{.new}` to the character that just [destroyed their body]{.new}. It is`

File: Omnibus.md [hunk 11]
```diff
@@ -591,84 +592,97 @@ attack the unsuspecting looters. If you are unsure as to whether someone's
...
+then that player's responsibility to present the [soul]{.new} token to an
```
Comment (pbobell) [inline Omnibus.md]: `then that player's responsibility to present the [soul token]{.new} to an`

File: Omnibus.md [hunk 12]
```diff
@@ -591,84 +592,97 @@ attack the unsuspecting looters. If you are unsure as to whether someone's
...
+considered [soulless]{.new}.
+
+### Fixing a [Soulless]{.new} Character ### {#fixing-a-soulless-character}
+
+To restore someone to life after [they have lost their soul]{.new},
+characters must first have the body of the person needing to be raised. Then
+characters must either cast a [Call the Soul](#call-the-soul), an
+[Intervention](#intervention) spell, or administer a
+[Potion of Soul Snare](#potion-of-soul-snare)
+(see [Strange Brew](#strange-brew)) to summon and reattach the soul. If
```
Comment (pbobell) [inline Omnibus.md]: `(see [Strange Brew](#strange-brew)) to summon and reattach the [soul]{.new}. If`

File: Omnibus.md [hunk 13]
```diff
@@ -591,84 +592,97 @@ attack the unsuspecting looters. If you are unsure as to whether someone's
...
+ticks is permanently dead, as their soul can no longer be restored by any means.
+On January 1st of each new year, one tick is removed from each PC that has any,
+unless they are already permanently dead.
+
+If a PC is killed and [rendered soulless]{.new} during an event, but
+returned to life before the end of the event, they do not get a tick.
+
+For a [body's complete destruction]{.new} to be official, it must be
+brought to the EH's attention. The EH must provide the Death Marshal with this
+information from  their events. A tick may only be issued or reported by a
```
Comment (pbobell) [inline Omnibus.md]: `information from their events. A tick may only be issued or reported by a`

Comment (pbobell) [reply Omnibus.md]: Removed an extra space between "from" and "their"

File: Omnibus.md [hunk 14]
```diff
@@ -591,84 +592,97 @@ attack the unsuspecting looters. If you are unsure as to whether someone's
...
+legal EH of the event  where the tick was incurred or by the player whose PC
```
Comment (pbobell) [inline Omnibus.md]: `legal EH of the event where the tick was incurred or by the player whose PC`

Comment (pbobell) [reply Omnibus.md]: Another extra space.

File: Omnibus.md [hunk 15]
```diff
@@ -2073,12 +2087,13 @@ is complete.
...
-An advanced regeneration is a regeneration from death [(]{.new}or scalping[, if
-specified in the spell description)]{.new} which begins when the character dies
-(or is scalped[,]{.new} if it brings a character back from scalping). Each time
-they die, the spellcaster may choose to double the length of their regeneration
-(to 240 seconds, or 480 if diseased). The regeneration is not considered used if
-the target is raised before their regeneration is complete.
+An advanced regeneration is a regeneration from death [(]{.new}or 
+[soul loss, if specified in the spell description)]{.new} which begins when 
+the character dies (or is [rendered soulless,]{.new} if it brings a character 
+back from [losing their soul]{.new}. Each time they die, the spellcaster may
```
Comment (pbobell) [inline Omnibus.md]: `back from [losing their soul)]{.new}. Each time they die, the spellcaster may `

File: Omnibus.md [hunk 16]
```diff
@@ -2243,6 +2258,7 @@ any).
 | [Intervention](#intervention)                        | [Resist Death](#resist-death)                        | [Transformation](#transformation)                    |
 +------------------------------------------------------+------------------------------------------------------+------------------------------------------------------+
 
+
```
Comment (pbobell) [inline Omnibus.md]: Why was this line added? Was it intentional?

File: Omnibus.md [hunk 17]
```diff
@@ -2376,13 +2393,13 @@ General](#animate-undead-general) spell were cast upon it, but the target's
...
-undead can be slain and raised or simply raised while undead to return to life.
-If a character under the effects of [Embrace Death](#embrace-death) has this
-cast on them, it acts as the spell [Animate Lesser
-Undead](#animate-lesser-undead) instead. You may have at most one undead general
-at a time.
+undead at the end of the event, that PC is considered dead and
+[soulless]{.new}. This spell will not work on a body without its
+[soul]{.new}. A character that is created as undead can be slain and raised
+or simply raised while undead to return to life. If a character under the
+effects  of [Embrace Death](#embrace-death) has this cast on them, it acts as
```
Comment (pbobell) [inline Omnibus.md]: `effects of [Embrace Death](#embrace-death) has this cast on them, it acts as`

Comment (pbobell) [reply Omnibus.md]: Removed extra space

File: Omnibus.md [hunk 18]
```diff
@@ -2376,13 +2393,13 @@ General](#animate-undead-general) spell were cast upon it, but the target's
...
+the spell  [Animate Lesser Undead](#animate-lesser-undead) instead. You may
```
Comment (pbobell) [inline Omnibus.md]: `the spell [Animate Lesser Undead](#animate-lesser-undead) instead. You may`

Comment (pbobell) [reply Omnibus.md]: Removed extra space

File: Omnibus.md [hunk 19]
```diff
@@ -2447,8 +2464,8 @@ utilize the special call with it. The weapon is one-handed, but may not be used
 in conjunction with another weapon or shield by the spellcaster. If the MC is IC
 broken, it will retain its magical properties if repaired.
 
-In addition, any scalping blow dealt by an Assassin's Blade counts as two blows
-dealt.
+In addition, any [body destroying blow]{.new} dealt by an Assassin's Blade
```
Comment (pbobell) [inline Omnibus.md]: `In addition, any [body destroying]{.new} blow dealt by an Assassin's Blade`

File: Omnibus.md [hunk 20]
```diff
@@ -2489,19 +2506,20 @@ corpse. The spell will end if the corpse reaches the spellcaster, the
...
 an opaque pouch or other way to keep them hidden - __Active:__ A quest may be
 required
 
-Allows the spellcaster to possibly find and reattach the soul of a scalped	
-character. When cast, the spellcaster presents the pouch of objects to the
-scalped character, who must then reach in and take one without looking. If the
-object is one of the four, the soul is reattached. If the object is the second
-color or design, nothing happens.This spell must be cast in the presence of the
-MM, who may require an additional quest be completed.
+Allows the spellcaster to possibly find and reattach the [soul]{.new} of a
```
Comment (pbobell) [inline Omnibus.md]: `Allows the spellcaster to possibly find and reattach the soul of a`

Comment (pbobell) [reply Omnibus.md]: Not a new instance of soul

File: Omnibus.md [hunk 21]
```diff
@@ -2489,19 +2506,20 @@ corpse. The spell will end if the corpse reaches the spellcaster, the
...
+[soulless]{.new} character. When cast, the spellcaster presents the pouch
+of objects to  the [soulless]{.new} character, who must then reach in and
```
Comment (pbobell) [inline Omnibus.md]: `of objects to the [soulless]{.new} character, who must then reach in and`

Comment (pbobell) [reply Omnibus.md]: Extra space

File: Omnibus.md (hunk 5 above)
Comment (ACannamela) [reply Omnibus.md]: Changed to:

A soul token]{.new} must have it's player's name and the PC's name 
written legibly on it.

Otherwise would have to be 

"Souls tokens must have players' names and PCs' names 
written legibly on them"

to keep number consistent.

File: Omnibus.md [hunk 22]
```diff
@@ -2759,34 +2778,35 @@ uses drama and theatrics has a better chance of success.
 ### Embrace Death (6th Circle) ### {#embrace-death}
 
 __Uses:__ 1 - __Verbal:__ 40 words - __Material:__ A container at least 4 inches
-in diameter able to hold the spellcaster's scalp token[, labeled with
+in diameter able to hold the spellcaster's [soul]{.new} token[, labeled with
```
Comment (3geek14) [inline Omnibus.md]: Okay, even though the old text was inconsistent about "scalp" vs. "scalp token", and this specific instance already had the word "token", I think "soul token" is the thing being added by this proposal, so I'd like to see "token" appear as new.

Comment (3geek14) [reply Omnibus.md]: The 2018 Omnibus uses the phrase "scalp token" all of fives times.  Four times in this spell and once in Regenerate the Scalp.  I'd like all five of those instances of "token" to be marked as new, but I'll concede if the OEC's majority disagrees.

File: Omnibus.md [hunk 23]
```diff
@@ -3052,13 +3075,15 @@ be moving their arms (e.g., parrying, so long as they don't step backwards).
...
-be under the effect of one Heartiness spell at a time. A use is considered to be
-over whenever the spellcaster receives at least 200 scalping blows, but is in
-effect until they are either fully scalped or raised.
+Having this spell makes [it harder to destroy the spellcaster's body]{.new}. 
+The next time the spellcaster['s body is destroyed]{.new} it will take 200 
+extra blows to successfully [destroy their body]{.new}. If struck for only 
+200 blows, instead of the full 400 blows, the spellcaster must inform the 
+[individual(s) destroying their body]{.new} that "The job is not yet done." 
+A spellcaster can only be under the effect of one Heartiness spell at a time. 
+A use is considered to be over whenever the spellcaster receives at least
```
Comment (3geek14) [inline Omnibus.md]: Out of scope, but I want to rant (and remember to write a proposal for next year) about how awful this spell description is.  A use is over when [EVENT] happens, but it's still in effect until [OTHER EVENT] happens.  WHEN DOES IT END?

HOW DO YOU (RE-)CAST THIS SPELL?  WHY DOES IT EVER END?

File: Omnibus.md [hunk 24]
```diff
@@ -3446,16 +3471,18 @@ Restore Enchantment
...
-### Regenerate the Scalp (5th Circle) ### {#regenerate-the-scalp}
+### Regenerate the [Soul]{.new} (5th Circle) ### {#regenerate-the-soul}
 
 __Uses:__ 1 - __Verbal:__ 30 words - __Material:__ An item similar to the
-spellcaster's scalp, with some sort of indication that it is their second scalp
-- __Caveats:__ [Regeneration](#regeneration-caveat), [Advanced
-Regeneration](#advanced-regeneration)
+spellcaster's [soul token]{.new}, with some sort of indication that it is
+their second [soul token]{.new} -
+__Caveats:__ [Regeneration](#regeneration-caveat),
```
Comment (3geek14) [inline Omnibus.md]: Not important, but `__Caveats:__` fits on the previous line.

File: Omnibus.md [hunk 25]
```diff
@@ -3446,16 +3471,18 @@ Restore Enchantment
...
+[Advanced Regeneration](#advanced-regeneration)
 
 This spell grants an advanced regeneration which [returns]{.new} a character to
-life from scalping [only]{.new}. After regenerating in this fashion, the
-material component becomes the spellcaster's new scalp token.
+life from [soullessness only]{.new}. After regenerating in this fashion, the
+material component becomes the spellcaster's new [soul]{.new} token.
```
Comment (3geek14) [inline Omnibus.md]: This is the one instance of "scalp token" in the old text that isn't in Embrace Death.

File: Omnibus.md [hunk 26]
```diff
@@ -591,84 +592,98 @@ attack the unsuspecting looters. If you are unsure as to whether someone's
...
-unless the PC is dead and scalped (see below), or unless they are under the
-influence of a spell which removes the PC's scalp. Scalps must have their
-player's name and the PC's name written legibly on it. A scalp does not exist
-physically in the IC world therefore it cannot be searched. A scalp is a
+A [soul token]{.new} is a token carried by each player which represents
+the life essence of that player's character. Players must be carrying their PC's
+[soul token]{.new} at all times unless the PC is dead and
+[soulless]{.new} (see [[Destroying a Body](#destroying-a-body) below), or 
+unless they are under the influence of a spell which removes the PC's 
+[soul. A soul token]{.new} must have it's player's name and the PC's name
```
[the remark this answers is not in the document]
Comment (pbobell) [reply Omnibus.md]: We should mark it as new, and remove the apostrophe. "must have it is player's name"?

File: Omnibus.md [hunk 27]
```diff
@@ -591,84 +592,98 @@ attack the unsuspecting looters. If you are unsure as to whether someone's
...
-player's name and the PC's name written legibly on it. A scalp does not exist
-physically in the IC world therefore it cannot be searched. A scalp is a
+A [soul token]{.new} is a token carried by each player which represents
+the life essence of that player's character. Players must be carrying their PC's
+[soul token]{.new} at all times unless the PC is dead and
+[soulless]{.new} (see [[Destroying a Body](#destroying-a-body) below), or 
+unless they are under the influence of a spell which removes the PC's 
+[soul. A soul token]{.new} must have it's player's name and the PC's name 
+written legibly on it. A [soul token]{.new} does not exist physically in the 
+IC world; therefore it cannot be searched. A [soul token]{.new} is a
```
[the remark this answers is not in the document]
Comment (pbobell) [reply Omnibus.md]: I'm on board with marking it as new. Might as well be as consistent as possible. (<--- Lots of "as" happening there.)

File: Omnibus.md [hunk 28]
```diff
@@ -591,84 +592,98 @@ attack the unsuspecting looters. If you are unsure as to whether someone's
...
-Brew](#strange-brew)) to summon and reattach the scalp. If characters lack the
-body, only an [Intervention](#intervention) spell will be able to return the
-scalped PC back to life. When the body and scalp are reunited, characters must
-cast one [Raise Dead](#raise-dead) spell for each event (including the first)
-since the scalped PC was scalped in order to raise them. A different spellcaster
-must provide each [Raise Dead](#raise-dead) spell used for this purpose.
+### [Destroying a Body]{.new} ### {#destroying-a-body}
+
+In order [for a player to separate a soul from a body, they must destroy
+it by striking]{.new} 200 blows with a weapon beside the body of the character
```
[the remark this answers is not in the document]
Comment (pbobell) [reply Omnibus.md]: I'm in favor of using Pi's suggested text. Good points in both cases, and within our power scope to fix.

File: Omnibus.md [hunk 29]
```diff
@@ -1058,10 +1073,10 @@ the middle of a fight. This form of attack deals no damage to armor.
 ### Impale ### {#impale}
 
 Impaling is the act of holding a weapon in an opponent's kill location after
-death, and saying the word "Impale." It counts as continuous, non-scalping blows
-to that location. The act of impaling will bypass any armor, damaged or not.
-While impaled, a character cannot be raised or animated and is unable to
-regenerate. The results can vary when impaling NPCs.
+death, and saying the word "Impale." It counts as continuous, [non-body
```
[the remark this answers is not in the document]
Comment (pbobell) [reply Omnibus.md]: Yep.

File: Omnibus.md [hunk 30]
```diff
@@ -2075,12 +2090,13 @@ is complete.
...
-An advanced regeneration is a regeneration from death [(]{.new}or scalping[, if
-specified in the spell description)]{.new} which begins when the character dies
-(or is scalped[,]{.new} if it brings a character back from scalping). Each time
-they die, the spellcaster may choose to double the length of their regeneration
-(to 240 seconds, or 480 if diseased). The regeneration is not considered used if
-the target is raised before their regeneration is complete.
+An advanced regeneration is a regeneration from death [(]{.new}or 
+[soul loss, if specified in the spell description)]{.new} which begins when 
+the character dies (or is [rendered soulless,]{.new} if it brings a character 
+back from [losing their soul)]{.new}. Each time they die, the spellcaster may
```
[the remark this answers is not in the document]
Comment (pbobell) [reply Omnibus.md]: Why?

File: Omnibus.md (hunk 22 above)
Comment (pbobell) [reply Omnibus.md]: I don't really care, but technically token should be black. I'll side with the third person to weigh in.

File: Omnibus.md (hunk 23 above)
Comment (pbobell) [reply Omnibus.md]: This is what happens when you try to make spells that are just skills or abilities.

File: Omnibus.md (hunk 25 above)
Comment (pbobell) [reply Omnibus.md]: Same as the other thread, fine either way.

File: Omnibus.md (hunk 26 above)
[the remark this answers is not in the document]
Comment (ACannamela) [reply Omnibus.md]: Adding both suggestions.

File: Omnibus.md (hunk 27 above)
[the remark this answers is not in the document]
Comment (ACannamela) [reply Omnibus.md]: Adding suggestion.

File: Omnibus.md (hunk 28 above)
[the remark this answers is not in the document]
Comment (ACannamela) [reply Omnibus.md]: Added suggestion

File: Omnibus.md [hunk 31]
```diff
@@ -591,84 +592,98 @@ attack the unsuspecting looters. If you are unsure as to whether someone's
...
+effectively divides the number of blows to be struck between the number of
+participants and number of weapons used. Some monsters and characters under
+certain spells may require more blows to completely [destroy]{.new}. If you
+strike 200 blows and the victim says "The job is not yet done," then the [body
+is not yet destroyed]{.new}. Some spells, such as
+[Strange Brew: Potion of Acid](#potion-of-acid) and
+[Assassin's Blade](#assassins-blade), may accelerate the act of [destroying a
+body]{.new}.
+
+After the [body is destroyed]{.new}, [the character whose body has been
```
[the remark this answers is not in the document]
Comment (ACannamela) [reply Omnibus.md]: Added

File: Omnibus.md (hunk 30 above)
[the remark this answers is not in the document]
Comment (ACannamela) [reply Omnibus.md]: Assuming you mean the second closing parentheses, agreed and added.

File: Omnibus.md (hunk 29 above)
[the remark this answers is not in the document]
Comment (ACannamela) [reply Omnibus.md]: Added

File: Omnibus.md (hunk 22 above)
Comment (ACannamela) [reply Omnibus.md]: It depends.  If we are using red to mark purely textual changes, then I think token should not be included (as the words were already there).

If we are saying that the phrase "soul token" is replacing the phrase "scalp token," I could see including token.  However, I think that opens another can of worms in having to reevaluate if other proposals change the words around them meaningfully enough to warrant including words that were already there as new text. I'm not necessarily against that, but it does seem a bit much to me.

I'm fine with whatever we do as long as we are consistent with implementing it.  I'm going to leave things as is for now, but may change it if we agree that the second case is how we want to handle things.

File: Omnibus.md (hunk 23 above)
Comment (ACannamela) [reply Omnibus.md]: I already made this joke to Pi, but I'm gonna leave it here as well.

If you only have heartiness, you have spells but can't cast any.  Does this mean you are not a spellcaster, but merely a spellhaver?

File: Omnibus.md (hunk 24 above)
Comment (ACannamela) [reply Omnibus.md]: Caveats moved to previous line, regeneration and advanced regeneration are broken up though (having both on the same line is 85 characters).

File: Omnibus.md (hunk 25 above)
Comment (ACannamela) [reply Omnibus.md]: See previous thread.

File: Omnibus.md (hunk 30 above)
[the remark this answers is not in the document]
Comment (3geek14) [reply Omnibus.md]: Right, I meant the second one.  `back from [losing their soul]{.new}).`

File: Omnibus.md [hunk 32]
```diff
@@ -626,7 +626,7 @@ is not yet destroyed]{.new}. Some spells, such as
 [Assassin's Blade](#assassins-blade), may accelerate the act of [destroying a
 body]{.new}.
 
-After the [body is destroyed]{.new}, [the character whose body has been
+After the [body is destroyed the character whose body has been
```
Comment (3geek14) [inline Omnibus.md]: The comma should've stayed between `destroyed` and `the`.

Comment (pbobell) [reply Omnibus.md]: Agreed. Removing the comma wasn't part of my suggested fix to this line.

Comment (ACannamela) [reply Omnibus.md]: Woops.

File: Omnibus.md (hunk 22 above)
Comment (ascerbus) [reply Omnibus.md]: Personally, I don't see the need the mark "token" as new (in the cases of scalp token --> soul token).  To me, the new marker is to assist in identifying actual changes in the text.  Marking the whole phrase could mislead people into thinking the word "token" didn't exist in the first place.
